pvr.octonet/pvr.octonet.txt

```diff
@@ -0,0 +1 @@
+pvr.octonet https://github.com/DigitalDevices/pvr.octonet master
```

Comment:
Please, point to a specific release tag here, not to master (or another branch).